### PR TITLE
Support Python 3.8

### PR DESCRIPTION
**Pull request by takano32**
[Image from Gyazo](https://gyazo.com/58d4d356576c21a56c54a501727b0f7d)

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -1 +1 @@
-3.6.9
+3.8.2
```
Comment (userlocalhost) [inline .python-version]: Could you please tell me the reason why you specify v3.8.2?
(When I install python3.8 package in the Ubuntu18.04 node, Python3.8.0 was installed)
スクリーンショット 2021-06-07 12 32 04

If you have any reason to install this version of Python, would you please write description how to install this version's Python on the Ubuntu18.04 in the README?

Comment (takano32) [reply .python-version]: This PR is not target mainly on Ubuntu 18.04 LTS, but Ubuntu 20.04 LTS.

Until recently, the version of Python on Ubuntu 20.04 LTS was 3.8.2.
It seems that the current version is 3.8.5, so i'll fix so.

```
takano32@atom:~$ cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.2 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

```
takano32@atom:~$ python3 -V
Python 3.8.5
```

```
takano32@atom:~$ dpkg --list |grep python3.8
ii  libpython3.8:amd64                   3.8.5-1~20.04.2                       amd64        Shared Python runtime library (version 3.8)
ii  libpython3.8-minimal:amd64           3.8.5-1~20.04.2                       amd64        Minimal subset of the Python language (version 3.8)
ii  libpython3.8-stdlib:amd64            3.8.5-1~20.04.2                       amd64        Interactive high-level object-oriented language (standard library, version 3.8)
ii  python3.8                            3.8.5-1~20.04.2                       amd64        Interactive high-level object-oriented language (version 3.8)
ii  python3.8-minimal                    3.8.5-1~20.04.2                       amd64        Minimal subset of the Python language (version 3.8)
```

Is it better targeted on Ubuntu 18.04 LTS?

Comment (takano32) [reply .python-version]: Now `.python-version` is `3.8.0`

File: requirements-dev.txt [hunk 2]
```diff
@@ -3,10 +3,10 @@ django-import-export==2.5.0
 django==2.2.20
 mock==2.0.0
 coverage>=4.4.1
-celery==4.2.0
+celery==4.4.7
 flower==0.9.3
 pyyaml==5.4
-kombu==4.2.1
+kombu==4.6.11
 python-memcached==1.59
```
Comment (userlocalhost) [inline requirements-dev.txt]: Is it necessary to update those libraries for adapting Python 3.8?

Comment (takano32) [reply requirements-dev.txt]: Celery 4.2.0 doesn't support Python 3.8.
And Celery 4.4.7 requires upper kombu version.

**Comment by takano32** [comment]
Celery Changelogs
* https://docs.celeryproject.org/en/v4.2.2/changelog.html
* https://docs.celeryproject.org/en/v4.3.0/changelog.html
* https://docs.celeryproject.org/en/v4.4.7/changelog.html

File: requirements-dev.txt (hunk 2 above)
Comment (takano32) [reply requirements-dev.txt]: Celery Changelogs
* https://docs.celeryproject.org/en/v4.2.2/changelog.html
* https://docs.celeryproject.org/en/v4.3.0/changelog.html
* https://docs.celeryproject.org/en/v4.4.7/changelog.html

**Review by userlocalhost — COMMENTED** [review]
Would you please update CHANGELOG to add description for this PR?

**Review by userlocalhost — APPROVED** [review]
LGTM 👍

**Pull request merged**